src/OSPSuite.R/Bootstrap/ApplicationStartup.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Threading;
+using OSPSuite.Core;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.Core.Serialization;
+using OSPSuite.Core.Serialization.Xml;
+using OSPSuite.Infrastructure;
+using OSPSuite.Infrastructure.Container.Castle;
+using OSPSuite.R.MinimalImplementations;
+using OSPSuite.Utility.Container;
+using IContainer = OSPSuite.Utility.Container.IContainer;
+
+namespace OSPSuite.R.Bootstrap
```

Comment:
This is the registration required to get the stuff going for R. Register Infrastructure, Core, Seriaization etc..